src/SDKs/CognitiveServices/dataPlane/Search/BingVisualSearch/BingVisualSearch/Microsoft.Azure.CognitiveServices.Search.VisualSearch.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Search.VisualSearch</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Image Visual Search API.</Description>
+    <VersionPrefix>1.1.0-preview</VersionPrefix>
```

Comment:
We strongly recommend starting versioning from `0.9.0-preview` for preview packages